_candidates_en/JulienTruffaut_1.html

```diff
@@ -0,0 +1,23 @@
+---
+name: Julien Truffaut
+title: Van Laarhoven Lens
+length: 40
+audience: Intermediate
+language: English
+twitter: JulienTruffaut
+github: julien-truffaut
+icon: http://www.gravatar.com/avatar/1d3a514afe5f1074ca145ec82eedf8fd.jpg
+organization:
+---
+<p>
+A Lens is a functional concept that permits to zoom in a data structure.
+A key point of Lenses is that they compose together such as you can drill deeper and deeper in your data.
+Once you arrive at the level you are interested in, you can either extract or modify (even effectfully!) the value you targeted.
+</p>
+<p>
+In this presentation, I would like to go through a specific Lens implementation known as Van Laarhoven Lenses.
+</p>
+<p>
+This implementation is particularly interesting because it defines Lenses as a unique function!
```

Comment:
原文: This implementation is particularly interesting because it defines Lenses as a unique function!
訳案1: この実装は 特に興味深いです。なぜならこれは Lens を単に1つの関数として定義できるからです。
訳案2: そしてこれは Lens を単に1つの関数として定義できる特に興味深い実装です。